src/base_object.h

```diff
@@ -40,7 +40,7 @@ class TransferData;
 
 class BaseObject : public MemoryRetainer {
  public:
-  enum InternalFields { kSlot, kInternalFieldCount };
+  enum InternalFields { kEmbedderType, kSlot, kInternalFieldCount };
 
   // Associates this object with `object`. It uses the 0th internal field for
```

Comment:
```suggestion
  // Associates this object with `object`. It uses the 1st internal field for
```
Since `kEmbedderType` was added to the start of the enum, `kSlot` would not be the `0th` index anymore, so this description should be updated.